README.md

```diff
@@ -47,3 +47,30 @@ team help ...
 # run jar
 java -jar target/github-stats-*.jar
 ```
+
+### CheckStyle
+
+```sh
+# compile the code
+mvn clean site
+```
+
+### Sonar
+
+Before running sonar:
+
+- Create a user
+- Register the project in sonar
+- Choose maven
+- Copy git code with credentials
```

Comment:
Where do we need to create a user and where do we have to _copy_ the code with which credentials?